src/core.c

```diff
@@ -4602,7 +4603,7 @@ static void EventThreadSpawn(char *device)
     fd = open(device, O_RDONLY | O_NONBLOCK);
     if (fd < 0)
     {
-        TRACELOG(LOG_WARNING, "RPI: Failed to open input device (error: %d)", device, worker->fd);
+        TRACELOG(LOG_WARNING, "RPI: Failed to open input device %s (error: %d)", device, errno);
```

Comment:
I try to avoid including additional libraries... could we just remove the error number information?

Reply:
The error number gives a hint why opening failed. But of course you can remove the include and write:
```
TRACELOG(LOG_WARNING, "RPI: Failed to open input device %s", device);
```